File: sample_programs/java_sample_programs/6cells.java
//import unwired.ApiException;
//import com.unwired.client.api.GeolocationApi;

public class GeolocationApiExample {

    public static void main(String[] args) {
        
     GeolocationApi apiInstance = new GeolocationApi();
    GeolocationSchema geolocationSchema = new GeolocationSchema(); // GeolocationSchema | 

    //CellSchema
    CellSchema c1=new CellSchema();
    c1.lac(7033);
    c1.cid(17811);

      CellSchema c1=new CellSchema();
    c1.lac(7033);
    c1.cid(17812);

      CellSchema c1=new CellSchema();
    c1.lac(7033);
    c1.cid(18513);

      CellSchema c1=new CellSchema();
    c1.lac(7033);
    c1.cid(16383);
    c1.signal(-60);
    c1.ta(13);

      CellSchema c1=new CellSchema();
    c1.lac(7033);
    c1.cid(12812);

      CellSchema c1=new CellSchema();
    c1.lac(7033);
    c1.cid(12811);

    List celldata = new ArrayList();
    celldata.add(c1);
    celladata.add(c2);
    celladata.add(c3);
    celladata.add(c4);
    celladata.add(c5);
    celladata.add(c6);

    geolocationSchema.token("YOUR_API_TOKEN");
    geolocationSchema.radio("gsm");
    geolocationSchema.mcc(310);
    geolocationSchema.mnc(404);
    geolocationSchema.cells(celldata);
    geolocationSchema.address(AddressSchema.NUMBER_1);

        try {
            Object result = apiInstance.geolocation(geolocationSchema);
            System.out.println(result);
        } catch (ApiException e) {
            System.err.println("Exception when calling GeolocationApi#geolocation");
            e.printStackTrace();
        }
    }
}

File: sample_programs/python_sample_programs/6cells.py
from __future__ import print_function
import time
import unwired
from unwired.rest import ApiException
from pprint import pprint

# create an instance of the API class
api_instance = unwired.GEOLOCATIONApi()

#cell schema
c1=unwired.CellSchema(lac=7033,cid=17811)
c2=unwired.CellSchema(lac=7033,cid=17812,signal=-60,ta=13)
c3=unwired.CellSchema(lac=7033,cid=18513)
c4=unwired.CellSchema(lac=7033,cid=16383)
c5=unwired.CellSchema(lac=7033,cid=12812)
c6=unwired.CellSchema(lac=7033,cid=12811) 
celldata=[c1,c2,c3,c4,c5,c6]

# GeolocationSchema
geolocation_schema = unwired.GeolocationSchema(token="YOUR_API_KEY",radio="gsm",mcc=310,mnc=404,cells=celldata,address=1)  

try:
    # Geolocation
    api_response = api_instance.geolocation(geolocation_schema)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling GEOLOCATIONApi->geolocation: %s\n" % e)
